src/server/debug.rs

```diff
@@ -1210,9 +1210,8 @@ fn divide_db(db: &Arc<DB>, parts: usize) -> raftstore::Result<Vec<Vec<u8>>> {
     let start = keys::data_key(b"");
     let end = keys::data_end_key(b"");
     let range = Range::new(&start, &end);
-    let region_id = 0;
     Ok(box_try!(
-        RocksEngine::from_db(db.clone()).divide_range(range, region_id, parts)
+        RocksEngine::from_db(db.clone()).get_range_approximate_split_keys(range, parts - 1)
```

Comment:
Why minus one?

Reply:
it provides the number of the parts, but what the function need to specify is number of split key (equals to parts - 1). Still can not make all the same.